TextFieldEffects/TextFieldEffects/HoshiTextField.swift

```diff
@@ -160,11 +185,7 @@ typealias VoidClosure = () -> Void
   // MARK: - TextFieldEffects
 
   override open func drawViewsForRect(_ rect: CGRect) {
-    frame = CGRect(x: frame.origin.x,
-                   y: frame.origin.y,
-                   width: rect.size.width ,
-                   height: height)
-    updateBorder()
+
```

Comment:
qq: if this method it's empty, do we still need it?

Reply:
sadly we do, it's required to be implemented, from its parent class

Reply:
suggestion: maybe just shorten the method body to one line `{ }`, or add a `//do nothing` comment?